### PR TITLE
Folder Added : Assignment 5 and Assignment 6

### PR DESCRIPTION
**Pull request by Jestinxavier**

### CODE REVIEW

**Review by vdongre304 — APPROVED** [review]

**Review by mubeen2710 — APPROVED** [review]

File: Jestin/HTML/Assignment5/home1.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    border: solid rgb(44, 44, 50);
+    border-width: 1px, 1px, 1px, 1px;
+    padding: 25px;
+    background-image: url("3.jpg");
+    opacity: 0.8;
+}
+.c {
+    font-family: Georgia, "Times New Roman", Times, serif;
+    text-align: right;
+}
```
Comment (Pawan1005) [inline Jestin/HTML/Assignment5/home1.css:26]: give proper names to class

File: Jestin/HTML/Assignment5/home1.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>css</title>
+        <link rel="stylesheet" href="home1.css" />
+    </head>
+    <body>
+        <div class="a">
```
Comment (Pawan1005) [inline Jestin/HTML/Assignment5/home1.html]: give proper names to class

File: Jestin/HTML/Assignment5/home1.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+                        <li><a href="MyHobby.html" target="_blank"> My Hobby</a></li>
+                        <li>
+                            <a href="PhotoGallery.html" target="_blank"> Photo Gallery</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </body>
+</html>
```
Comment (Pawan1005) [inline Jestin/HTML/Assignment5/home1.html:34]: give proper names to class

File: Jestin/HTML/Assignment6/Ass5q6.css [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    background-color: blue;
+    border-radius: 2px 8px;
+}
+.c {
+    font-family: Arial, Helvetica, sans-serif;
+}
+::placeholder {
+    color: gray;
+    opacity: 0.6;
+}
```
Comment (Pawan1005) [inline Jestin/HTML/Assignment6/Ass5q6.css:19]: give proper names to class

File: Jestin/HTML/Assignment6/Ass5q6.html [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
+        <link rel="stylesheet" href="Ass5q6.css" />
+    </head>
+    <body class="c">
+        <h2 class="a">MI-9 Intern HTML Assignment</h2>
+        <div>
+            <hr />
+        </div>
```
Comment (Pawan1005) [inline Jestin/HTML/Assignment6/Ass5q6.html]: no need of div tag here

File: Jestin/HTML/Assignment6/Ass5q6.html [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+        <div>
+            <h4 class="b">personal Details</h4>
```
Comment (Pawan1005) [inline Jestin/HTML/Assignment6/Ass5q6.html]: give proper names to class

File: Jestin/HTML/Assignment6/Ass5q6.html [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+                <input type="radio" id="less than 18" name="less than 18" />
+                <label for="less than 18">less than 18</label>
+                <input type="radio" id="18-25" name="18-25" checked />
+                <label for="18-25">18-25</label>
+                <input type="radio" id="26-40" name="26-40" />
+                <label for="26-40">26-40</label>
+            </form>
+        </div>
+        <div>
+            <h4 class="b">Academic Details</h4>
```
Comment (Pawan1005) [inline Jestin/HTML/Assignment6/Ass5q6.html]: give proper names to class

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**